pkg/compose/start.go

```diff
@@ -115,7 +115,7 @@ func (s *composeService) watchContainers(ctx context.Context, projectName string
 				restarted := watched[container.ID]
 				watched[container.ID] = restarted + 1
 				// Container terminated.
-				willRestart := willContainerRestart(inspected, restarted)
+				willRestart := !s.stopping && willContainerRestart(inspected, restarted)
```

Comment:
This will for sure stop compose for watching, but container will still run!

Reply:
Oh no, my bad. `willContainerRestart` tells us container is _configured_ to restart, but doesn't guarantee it will.
Anyway we can't just stop this way, as this just assumes container will stop as we asked him to, but doesn't check.